transformers/events/tend/transformer_test.go

```diff
@@ -44,11 +44,18 @@ var _ = Describe("Tend transformer", func() {
 			models, err := transformer.ToModels(constants.FlipABI(), []core.EventLog{test_data.TendEventLog}, db)
 			Expect(err).NotTo(HaveOccurred())
 
-			var addressID int64
+			var addressID, msgSenderAddressID int64
+
 			addrErr := db.Get(&addressID, `SELECT id FROM public.addresses`)
```

Comment:
This line looks like it happens to depend on the order, so it gets the right address by luck.